src/main/webapp/dashboard.xhtml

```diff
@@ -123,6 +124,31 @@
                                     </div>
                                 </div>
                             </div>
+                            <div class="col-md-4">
+                                <div class="card-box">
+                                    <h4 class="header-title">#{bundle['dashboard.card.licenses']}</h4>
+                                    <div class="row text-center">
+                                        <div class="col-xs-6">
+                                            <h:outputText
+                                                    value="#{LicenseListingPage.activeAndInactiveLicensesCount._1()}"/>
+                                            <p class="small text-muted">#{bundle['dashboard.card.licenses.active']}</p>
+                                        </div>
+                                        <div class="col-xs-6">
+                                            <h:outputText
+                                                    value="#{LicenseListingPage.activeAndInactiveLicensesCount._2()}"/>
```

Comment:
I think that we shouldn't mix view pages. I would move responsibility of counts to DashboardPage